00-Raster-Structure.Rmd

```diff
@@ -353,44 +346,107 @@ GDALinfo("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
 
 ```
 
-## Bad data values
-
-Bad data values are different from NoData values. Sometimes a raster's metadata
-will tell us a value range of values for the raster. Sometimes, we need to use
-some practial scientific insight as we examine the data - just as we would for field
-data. 
+2. **Bad data values:** Bad data values are different from NoData values. 
```

Comment:
I could add a geotiff with some REALLY REALLY high values that don't make sense?? we could assign those to NaN? Thoughts?